pilz_trajectory_generation/src/trajectory_generator_circ.cpp

```diff
@@ -251,12 +169,42 @@ bool TrajectoryGeneratorCIRC::extractMotionPlanInfo(const planning_interface::Mo
 
   info.circ_path_point.first = req.path_constraints.name;
   info.circ_path_point.second = circ_path_point;
+}
+
+void TrajectoryGeneratorCIRC::plan(const planning_interface::MotionPlanRequest &req,
+                                   const MotionPlanInfo& plan_info,
+                                   const double& sampling_time,
+                                   trajectory_msgs::JointTrajectory& joint_trajectory)
+{
+  // create Cartesian path for circle
+  std::unique_ptr<KDL::Path> path(setPathCIRC(plan_info));
```

Comment:
Maybe remove comment and rename `path` into `cart_path`?

Reply:
Done.